test.sh

```diff
@@ -0,0 +1,15 @@
+ISSUES_MANAGER_VERSION=$(xpath -q -e  '/project/version/text()' pom.xml)
```

Comment:
Is this file still relevant? It uses the old name and the old-style jars…

Reply:
If possible, we should remove or replace it like also to use xpath which is not a tool installed on all the OS.

Reply:
oups!! no, no, I pushed it by mistake. I will remove it

Reply:
This file needs to be removed.

Reply:
:-S
Done!